src/components/network-modification-table/renderers/description-cell.tsx

```
@@ -0,0 +1,83 @@
+                    <IconButton
+                        onClick={handleModifyDescription}
+                        disabled={isLoading || isDisabled}
+                        sx={createEditDescriptionStyle(data.description)}
+                    >
+                        <EditNoteIcon empty={empty} />
+                    </IconButton>
+                </span>
+            </Tooltip>
+        </>
+    );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)